docs/troubleshooting/clusterResourcePlacementOverridden.md

```diff
@@ -0,0 +1,156 @@
+  resourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: secret-reader
+      version: v1
+  policy:
+    placementType: PickN
+    numberOfClusters: 1
+    affinity:
+      clusterAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: canary
+  strategy:
+    type: RollingUpdate
+    applyStrategy:
+      allowCoOwnership: true
+```
+
+### CRP Status:
+```
+status:
+  conditions:
+  - lastTransitionTime: "2024-05-14T16:16:18Z"
+    message: found all cluster needed as specified by the scheduling policy, found
+      1 cluster(s)
+    observedGeneration: 1
+    reason: SchedulingPolicyFulfilled
+    status: "True"
+    type: ClusterResourcePlacementScheduled
+  - lastTransitionTime: "2024-05-14T16:16:18Z"
+    message: All 1 cluster(s) start rolling out the latest resource
+    observedGeneration: 1
+    reason: RolloutStarted
+    status: "True"
+    type: ClusterResourcePlacementRolloutStarted
+  - lastTransitionTime: "2024-05-14T16:16:18Z"
+    message: Failed to override resources in 1 cluster(s)
+    observedGeneration: 1
+    reason: OverriddenFailed
+    status: "False"
+    type: ClusterResourcePlacementOverridden
+  observedResourceIndex: "0"
+  placementStatuses:
+  - applicableClusterResourceOverrides:
+    - cro-1-0
+    clusterName: kind-cluster-1
+    conditions:
+    - lastTransitionTime: "2024-05-14T16:16:18Z"
+      message: 'Successfully scheduled resources for placement in kind-cluster-1 (affinity
+        score: 0, topology spread score: 0): picked by scheduling policy'
+      observedGeneration: 1
+      reason: Scheduled
+      status: "True"
+      type: Scheduled
+    - lastTransitionTime: "2024-05-14T16:16:18Z"
+      message: Detected the new changes on the resources and started the rollout process
+      observedGeneration: 1
+      reason: RolloutStarted
+      status: "True"
+      type: RolloutStarted
+    - lastTransitionTime: "2024-05-14T16:16:18Z"
+      message: 'Failed to apply the override rules on the resources: add operation
+        does not apply: doc is missing path: "/metadata/labels/new-label": missing
+        value'
+      observedGeneration: 1
+      reason: OverriddenFailed
+      status: "False"
+      type: Overridden
+  selectedResources:
+  - group: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: secret-reader
+    version: v1
+```
+The CRP attempted to override a propagated resource utilizing an applicable `ClusterResourceOverrideSnapshot`.
+However, as the `ClusterResourcePlacementOverridden` condition remains false, looking at the placement status for the cluster
+where the condition `Overriden` failed will offer insights into the exact cause of the failure.
+The accompanying message highlights that the override failed due to the absence of the path `/metadata/labels/new-label` and its corresponding value.

```

Comment:
Could we please spell it out, when we use metadata/labels/new-label we are trying to modify/add a label with the value we provide if the labels field is not empty. But in the case /metadata/labels we are replacing the field with the value we provide